packages/Frontend/src/components/SignIn.jsx

```diff
@@ -1,19 +1,40 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
+import { Button, Divider, Form, Header, Icon, Loader, Segment, Message } from 'semantic-ui-react';
 
 export default function SignIn({
   error, loading, signIn, submitIfEnter, updateFormByKey,
 }) {
   return (
-    <Fragment>
-      <p>Username: </p>
-      <input onChange={updateFormByKey('username')} onKeyUp={submitIfEnter} />
-      <p>Password: </p>
-      <input onChange={updateFormByKey('password')} onKeyUp={submitIfEnter} type="password" />
-      <button onClick={signIn}>Submit</button>
-      { loading ? <p>Loading...</p> : null }
-      { error ? <p>{error}</p> : null }
-    </Fragment>
+    <div style={{
+        margin: 'auto',
+        position: 'relative',
+        top: 100,
+        width: '50%',
+      }}
+    >
+      <Segment>
+        <Header as="h1" icon textAlign="center">
+          <Icon name="user" circular />
+          <Header.Content>
+            Sign In
+          </Header.Content>
+        </Header>
+        <Form>
+          <Form.Input placeholder="Username" onChange={updateFormByKey('username')} onKeyUp={submitIfEnter} />
+          <Form.Input placeholder="Password" onChange={updateFormByKey('password')} onKeyUp={submitIfEnter} type="password" />
+          <Button color="violet" onClick={signIn} fluid>Sign In</Button>
+          <Divider horizontal>Or</Divider>
+          <Button color="blue" fluid>Sign Up Now</Button>
+        </Form>
+        { loading && <Loader active inline /> }
```

Comment:
I don't think the Loader belongs here and/or set up this way. Look at semantic ui's Loader's and see if there's a better looking Loader. You can try to sign in and it'll show the Loader for a second if you want to see what it looks like, or you can use react dev tools and go and set loading to be true, or just change it to be true in code while you're working on it.